src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -319,16 +322,6 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
     };
 
     if (currentNode.type === NodeType.TableArtifact) {
-      // Since workflow is pending, it doesn't have a result set yet.
```

Comment:
I think I understand, but to double check — why is this code no longer necessary?

Reply:
This is where I'm deprecating the export csv button.